### DISCUSSION TITLE
Removing button breaks homepage

### DISCUSSION THREAD

**Discussion by BlancaMorillo** [thuliteio/doks#974] [Q&A] [answered]
Hello! 

I am working with the homepage, once removing the get-started button in line 9 in ```layouts/index.html``` and run ```npm run build```, it breaks everything. I have been looking if someone has faced something similar but have yet not found anything.

<img width="1688" alt="image" src="https://user-images.githubusercontent.com/25622730/210115719-6dcbb79e-68c5-4411-83b2-3cb218b23ca9.png">

Thank you so much for any help!

**Comment by h-enk** [comment, marked as answer]
Have a repo you could share?

**Reply by BlancaMorillo** [reply]
Sure, here it is :) [Link](https://github.com/BlancaMorillo/test )

**Reply by h-enk** [reply]
In `./config/_default/config.toml` set `baseurl = "/"`. 

Preview your site locally by e.g. running [http-server](https://github.com/http-party/http-server) (included in next Doks release, so you can then just run `npm run preview`) — first install http-server locally in your project: `npm I http-server -D`, then run `npx http-server`.

**Reply by BlancaMorillo** [reply]
Thank you for your help! I did the change of the baseurl in `./config/_default/config.toml `and `./config/production/config.toml`, it worked using the http-server.

But when I run`npm run build` and look for `.public/index.html`, it still shows me the same as before 🤔 .

**Reply by h-enk** [reply]
> But when I run `npm run build` and look for `.public/index.html`, it still shows me the same as before 🤔 .

Correct, that's to be expected.

When you open Developer Tools (F12 in Chrome), you'll see a number of errors/reasons why (related to CORS headers, but really because of files not found).  You'll need to serve the files to make it work.

**Reply by BlancaMorillo** [reply]
Thank you so much h-enk, I will mark this as closed :)
